lib/versioning/VersionID.js

```diff
@@ -116,12 +116,17 @@ function generateVersionId(info, replicationGroupId) {
     lastSeq = (lastTimestamp === ts) ? lastSeq + 1 : 0;
     lastTimestamp = ts;
 
+    // if USE_HEX_VERSION_ID is set, info is used. By default, it is not used.
+    if (process.env.USE_HEX_VERSION_ID === 'no' || !process.env.USE_HEX_VERSION_ID) {
```

Comment:
better name for this?

Reply:
as discussed, I suggest checking for `process.env.S3_VERSION_ID_ENCODING_TYPE=hex` specifically